shared/src/packages/chats/libs/validation-schemas/create-chat.validation-schema.ts

```diff
@@ -0,0 +1,14 @@
+import joi from 'joi';
+
+import { ChatValidationMessage } from '../enums/enums.js';
+import { type ChatRequestDto } from '../types/types.js';
+
+const createChat = joi.object<ChatRequestDto, true>({
+  name: joi
+    .string()
+    .trim()
+    .required()
+    .messages({ 'any.required': ChatValidationMessage.NAME_REQUIRED }),
```

Comment:
Also 'string.required' and 'string.empty' is needed in order to avoid situations with entered space e.t.c.

Reply:
thanks, will add this

Reply:
do we also need `string.required` when we have `any.required`?

Reply:
there is no `string.required` according to joi docs

Reply:
Yeah, I missed that, I got confused with another message type. So adding only `string.empty` will be enough

Reply:
done